frontend/components/site/siteBuildLogs.jsx

```diff
@@ -26,6 +52,11 @@ const SiteBuildLogs = () => {
           <li><DownloadBuildLogsButton buildId={buildId} buildLogsData={logs} /></li>
         </ul>
       </div>
+      {buildTooOld(buildDetails) && (
```

Comment:
```{buildTooOld(buildDetails) && (```
 and its contents could be moved inside of the closure for
```{(!logs || logs?.length === 0) && (```
on line 60, with 
```<SiteBuildLogTable buildLogs={['This build does not have any build logs.']} />```
only showing when `buildTooOld(buildDetails)` is false